### ISSUE TITLE
FYI: SignalR-Web-Client

### ISSUE THREAD

**Issue by enricop** [Dorin-Mocan/SignalRSwaggerGen#16]
At the link below there is a great tool to test SignalR hubs from the browser.
https://github.com/gourav-d/SignalR-Web-Client

A very cool feature would be automatically gerating such "web-client" for each hub methods.

thanks

**Comment by Dorin-Mocan** [comment]
@enricop , hello my friend. I'll take a look at the issues you raised and come back to you. Thank you 🙏

**Comment by ArturMarekNowak** [comment]
@Dorin-Mocan your Nuget is really cool but adding mentioned by @enricop functionality would make it perfect 😍

**Comment by Dorin-Mocan** [comment]
@ArturMarekNowak , yes, the tool is pretty cool, but I need to investigate on how to integrate it with my package.

**Comment by Dorin-Mocan** [comment]
@ArturMarekNowak  @enricop , at the moment I don't see any way of integrating my SignalRSwaggerGen with SignalR Web Client other than serializing hub data in a file and let it be consumed by the web client. But for this we need changes on both products. If the creator of SignalR Web Client agrees to participate in this integration, then fine by me.
